GW2EIEvtcParser/EncounterLogic/Raids/W4/Bosses/MursaatOverseer.cs

```diff
@@ -134,6 +134,28 @@ internal override void ComputePlayerCombatReplayActors(PlayerActor player, Parse
         replay.Decorations.AddOverheadIcons(claims, player, ParserIcons.FixationPurpleOverhead);
     }
 
+    internal override void ComputeEnvironmentCombatReplayDecorations(ParsedEvtcLog log, CombatReplayDecorationContainer environmentDecorations)
+    {
+        base.ComputeEnvironmentCombatReplayDecorations(log, environmentDecorations);
+
+        (long start, long end) lifespan;
+
+        // Protect - Bubble
+        if (log.CombatData.TryGetEffectEventsByGUID(EffectGUIDs.MursaarOverseerProtectBubble, out var protects))
```

Comment:
Aren't the projectile and bubble effect sourced from players? If yes it would make more sense to move it to PlayerDecorations

Reply:
Didn't see it as necessary but I've moved them